services/liquibase/main-db/changelog/changelogs/2025-04-19-1840-init-schema.yaml

```diff
@@ -0,0 +1,170 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-full-schema
+      author: wojciakjakub
+      changes:
+      
+        # recipes
+        - createTable:
+            tableName: recipes
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: name, type: varchar(100) }
+              - column: { name: description, type: varchar(2000) }
+
+        # ingredients
+        - createTable:
+            tableName: ingredients
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: name, type: varchar(100) }
+
+        # ingredients_categories
+        - createTable:
+            tableName: ingredients_categories
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: name, type: varchar(100) }
+              - column: { name: description, type: varchar(1000) }
+
+        # ingredients_categories_connection
+        - createTable:
+            tableName: ingredients_categories_connection
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: ingredient_id, type: uuid }
+              - column: { name: tag_id, type: uuid }
+
+        # recipe_ingredients
+        - createTable:
+            tableName: recipe_ingredients
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: recipe_id, type: uuid }
+              - column: { name: ingredient_id, type: uuid }
+
+        # users
+        - createTable:
+            tableName: users
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: first_name, type: varchar(200) }
+              - column: { name: second_name, type: varchar(200) }
+              - column: { name: last_name, type: varchar(200) }
+              - column: { name: email_address, type: varchar(200) }
+              - column: { name: password_hash, type: varchar(512) }
+
+        # user_fridge_ingredients
+        - createTable:
+            tableName: user_fridge_ingredients
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: user_id, type: uuid }
+              - column: { name: ingredient_id, type: uuid }
```

Comment:
ingredient id should be a foreign key pointing to the ingredients table, field id